features/works/chapter_edit.feature

```diff
@@ -366,9 +367,31 @@ Feature: Edit chapters
       And I should see "Chapter by originalposter"
 
 
+  Scenario: Removing yourself as a co-creator from last remaining co-created chapter
```

Comment:
I think this scenario and "Removing yourself as a co-creator from the chapter edit page" can be combined.